benchmark/torch/NeurIPS2020-Learning-to-Run-a-Power-Network-Challenge/track2/evaluate.py

```diff
@@ -0,0 +1,161 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import os
+from grid2op.Runner import Runner
+from l2rpn_baselines.utils.save_log_gif import save_log_gif
+from rl_agent import RLAgent
+import argparse
+
+
+def cli():
+    parser = argparse.ArgumentParser(description="Eval baseline RLAgent")
+    parser.add_argument(
+        "--load_path",
+        default='./saved_files',
+        help="The path to the model [.h5]")
+    parser.add_argument(
+        "--logs_path",
+        required=False,
+        default='./logs_path',
+        type=str,
+        help="Path to output logs directory")
+    parser.add_argument(
+        "--nb_episode",
+        required=False,
+        default=1,
+        type=int,
+        help="Number of episodes to evaluate")
+    parser.add_argument(
+        "--nb_process",
+        required=False,
+        default=1,
```

Comment:
Have you tested running the script with nb_process > 1?

Reply:
tested.